apps/src/blockly/googleBlocklyWrapper.ts

```diff
@@ -462,6 +475,41 @@ function initializeBlocklyWrapper(blocklyInstance: GoogleBlocklyInstance) {
     return this.fieldRow;
   };
 
+  /**
+   * Enable the specified field helper with the specified options for this
+   * input's connection
+   * @param {string} fieldHelper the field helper to retrieve. One of
+   *        Blockly.BlockFieldHelper
+   * @param {*} options for this helper
+   * @return {!Blockly.Input} The input being modified (to allow chaining).
+   */
+  extendedInput.addFieldHelper = function (
+    fieldHelper: string,
+    options: FieldHelperOptions
+  ) {
+    // if (this.type !== Blockly.INPUT_VALUE) {
```

Comment:
remove this?

Reply:
Oh, thanks! I meant for this to be here, but I needed to correct the way the type check was written.